test/Directory.Build.props

```diff
@@ -0,0 +1,23 @@
+<Project>
+  <Import Project="..\Directory.Build.props" />
```

Comment:
Right. The Directory.Build import only finds the first file in or above the project directory. We have to manually chain beyond that.